### DISCUSSION TITLE
Mac OSX support

### DISCUSSION THREAD

**Discussion by heboth** [ravendb/ravendb#13031] [Q&A] [answered]
Hi all,

I'm trying to install RavenDB on my Mac OS (Monterey) but it seems I can not download a version.  However, the documentation, states that Mac OS X is supported.  Does anyone have any info it works or how I can make it work?

from the documentation is says

RavenDB is cross-platform with support for the following operating systems:

_Windows x64 / x86
Linux x64
Docker
macOS
Raspberry Pi_

On the download page, it says

_PLATFORM
 WINDOWS
 LINUX
 RASPBERRY
 DOCKER_

Linux version will not run on my Mac OS :(

Br

Henrik

**Comment by haludi** [comment, marked as answer]
Hi,

You can download it from here -
https://hibernatingrhinos.com/builds?product=RavenDB&version=All+versions&channel=Stable

Best regards,
Igal

**Reply by heboth** [reply]
Thanks Igal

**Reply by haludi** [reply]
Hi,

Sorry, but I was updated that there is a current open issue with MacOS.
Should be solved soon - https://issues.hibernatingrhinos.com/issue/RavenDB-16198.

Best regards,
Igal
